katas/square-digits/jeanpeare/squareDigits.js

```diff
@@ -1,3 +1,12 @@
 export function squareDigits(num) {
-    return 0;
+    let divideArr = [...`${num}`].map(c => parseInt(c)); // Divide input number in an array
+    var i = divideArr.length;                            // Get the length of the Array created
```

Comment:
Please use meaningful names instead of `i` and `j`. Based on what they seem to contain, I'd rename them to something like `arrayLength` and `arrayIndex` or something similar